avocado/core/runner.py

```diff
@@ -217,7 +217,7 @@ def _run_test(self, test_factory, queue):
         :param test_factory: Test factory (test class and parameters).
         :type test_factory: tuple of :class:`avocado.core.test.Test` and dict.
         :param queue: Multiprocess queue.
-        :type queue: :class`multiprocessing.Queue` instance.
+        :type queue: :class:`multiprocessing.Queue` instance.
```

Comment:
This is quite common, maybe we can add card to review and fix this kind of mistake...